### PR TITLE
Added the latest one that just launched

### PR DESCRIPTION
**Pull request by panaalexlateral**
Added a nice spin to the original game.

Here's a promo code for you : https://phobos.apple.com/WebObjects/MZFinance.woa/wa/freeProductCodeWizard?code=JE6MYK766677

Have fun playing!

### CODE REVIEW

File: _data/variants.yml [hunk 1]
```diff
@@ -1,7 +1,11 @@
-- url: http://gabrielecirulli.github.io/2048/
+﻿- url: http://gabrielecirulli.github.io/2048/
```
Comment (agelber) [inline _data/variants.yml]: What exactly changed here? Is this just Github being weird?

**Comment by agelber** [comment]
Thanks! But also, please fix your indentation and move your addition down to the bottom of the list :grin:

File: _data/variants.yml (hunk 1 above)
Comment (panaalexlateral) [inline _data/variants.yml]: no change here, weird

**Comment by panaalexlateral** [comment]
Fixed.

Thanks,
Alex.

**Comment by agelber** [comment]
:cool::cool::cool:

**Pull request merged**